### PR TITLE
landing page

### LINKED ISSUE #1: Landing Page
### Home page
- [x] Banner
- [x] Deals
- [x] Featured categories
- [x] Featured brands
- [x] Footer

### PR DESCRIPTION
**Pull request by siddhigate**
### Landing page
- Banner
- Featured deals, categories, brands

**Live preview:** https://fashionly-ui-git-landing-page-siddhigate.vercel.app/pages/landing.html

landing

### CODE REVIEW

**Review by BelideNikhil — APPROVED** [review]
One quick suggestion is to either compress the image size or use webp format images as a good practice.

The UI is looks great.
GTG!

**Review by srishti-maurya — COMMENTED** [review]
looks good!
suggestion: your utility class names could be more descriptive

**Comment by siddhigate** [comment]
> 

Thanks for your review, Nikhil! Done with changes.

**Comment by siddhigate** [comment]
> looks good! suggestion: your utility class names could be more descriptive

Thanks for you review, Srishti! Have changed the name of the utility classes.

**Comment by siddhigate** [comment]
> One quick suggestion is to either compress the image size or use webp format images as a good practice.
> 
> The UI is looks great. GTG!

Thanks for your review, Nikhil! Made the changes accordingly.

**Review by gopalbbharadva — APPROVED** [review]

File: css/pages/footer.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.footer {
+  min-height: 70vh;
+  background: #f9fafb;
```
Comment (ZuberDunge) [inline css/pages/footer.css]: you can use css variables here.

**Review by ZuberDunge — COMMENTED** [review]
Very good code structure, clean & readable. UI looks pretty too, Good work Siddhi.  

Good to go from my side.

**Review by ZuberDunge — APPROVED** [review]

File: css/pages/footer.css (hunk 1 above)
Comment (siddhigate) [reply css/pages/footer.css]: Thanks for pointing it out!

**Pull request merged**